test/fixtures/test-helpers/input.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  app: {
+    'app.js': '// app',
+    'resolver.js': '// resolver'
+  },
+  config: {
+    'environment.js': '"ENV"'
+  },
+  tests: {
+    helpers: {
+      'resolver.js': 'import Resolver from "../../app/resolver";',
```

Comment:
I take that back - it doesn't currently transform correctly. But still, which of `../../app/resolver` and `../../resolver` are correct?